DQMOffline/Trigger/plugins/TopMonitor.cc

```diff
@@ -574,7 +619,22 @@ void TopMonitor::analyze(edm::Event const& iEvent, edm::EventSetup const& iSetup
   }
   if ( muons.size() < nmuons_ ) return;
 
+    //george
+
+  if (nmuons_>1){
+    TLorentzVector mu1,mu2;
+    mu1.SetPtEtaPhiM(muons.at(0).pt(),muons.at(0).eta(),muons.at(0).phi(),0.105);
+    mu2.SetPtEtaPhiM(muons.at(1).pt(),muons.at(1).eta(),muons.at(1).phi(),0.105);
+    mll=(mu1+mu2).M();
+    sign=muons.at(0).charge()*muons.at(1).charge();
+    }
+  if (nmuons_>1 && invMassUppercut_>-1 &&  invMassLowercut_>-1 && (mll>invMassUppercut_ || mll<invMassLowercut_)) return;
+ if (nmuons_>1 && opsign_ && sign==1) return;  
+
+  //cout<<" mll="<<mll<<"  invMasscut_="<<invMasscut_<<endl;
   double eventHT = 0.;
+  TVector3 eventMHT;
```

Comment:
please make this a math::XYZVector so you can avoid the copies below and just do 

eventMHT += j.momentum()